replica/replica.go

```diff
@@ -237,6 +237,9 @@ func construct(preload, readonly bool, size, sectorSize int64, dir, head string,
 	}
 	r.info.Parent = r.diskData[r.info.Head].Parent
 
+	if err = r.removeStaleFromChildrenMap(); err != nil {
```

Comment:
why do we want to add and then to remove? can't we avoid adding in first place?

Reply:
readMetadata() func adds them to list, till that point chain is not formed. Chain is formed in openLiveChain() which is called later.